src/styles/sidebar.scss

```diff
@@ -69,9 +70,9 @@
     &__items {
         list-style: none;
         padding: 0;
-        margin-top: 8px;
-        padding-top: 2px;
-        transition: max-height 0.5s ease;
+        margin-top: 0px;
+        padding-top: 4px;
```

Comment:
Changing the `margin` and `padding` here will change the spacing in the vertical sidebar. Please keep the same aspect.